Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1749,18 +1752,87 @@ namespace AtomSampleViewer
 
         if (xrSystem)
         {
+            RHI::Ptr<RHI::Device> device = Utils::GetRHIDevice();
+            if (RHI::CheckBitsAll(device->GetFeatures().m_shadingRateTypeMask, RHI::ShadingRateTypeFlags::PerRegion) && !m_vrsTexture)
+            {
+                // Need to fill the contents of the Variable shade rating image.
+                const AZStd::shared_ptr<const RPI::PassTemplate> forwardTemplate =
+                    RPI::PassSystemInterface::Get()->GetPassTemplate(Name("MultiViewForwardPassTemplate"));
+                // Find the Shading Rate Attachment
+                Data::Asset<RPI::AttachmentImageAsset> vrsImageAsset;
+                for (const auto& imageAttachment : forwardTemplate->m_imageAttachments)
```

Comment:
Maybe not something we need to solve now but what happens if there are multiple passes with different shading rate attachments? At the moment only the first one will get updated.